fast64_internal/oot/oot_skeleton.py

```diff
@@ -81,13 +81,25 @@ def toC(self):
 				str(self.getNumLimbs()) + " };\n\n" 
 			data.header = "extern SkeletonHeader " + self.name + ";\n"
 
+		index = 0
+		for limb in limbList:
+			name = (self.name + "_" + toAlnum(limb.boneName)).upper()
+			if limb.index == 0:
+				data.header += "#define " + name + "_POS_LIMB 0\n"
+				data.header += "#define " + name + "_ROT_LIMB 1\n"
+			else:
+				data.header += "#define " + name + "_LIMB " + str(limb.index+1) + "\n"
+			index += 1
+		data.header += "#define " + self.name.upper() + "_NUM_LIMBS " + str(index+1) + "\n"
```

Comment:
you can remove `index` and
```suggestion
		data.header += "#define " + self.name.upper() + "_NUM_LIMBS " + str(len(limbList)+1) + "\n"
```

then is this really _num_limbs or _frame_table_length, what do you intend this define as